compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -855,6 +858,25 @@ internal class RootNodeOwner(
             snapshotInvalidationTracker.requestDraw()
         }
 
+        override val isArrEnabled: Boolean = @OptIn(ExperimentalComposeUiApi::class) ComposeUiFlags.isAdaptiveRefreshRateEnabled
+        private var currentFrameRate = Float.NaN
+
+        override fun voteFrameRate(frameRate: Float) {
+            if (isArrEnabled) {
```

Comment:
Let's convert it to precondition to reduce indention
```suggestion
            if (!isArrEnabled) return
```